docs/hotspot-makers/5g-hardware-specification.mdx

```diff
@@ -0,0 +1,211 @@
+| USB 3.0        | Required for imaging process                                                       |
+| Software       | Rust, Python, Magma (check Magma dependencies)                                     |
+
+\* FreedomFi partners only. TPM 2.0 not required for non-Freedom Fi partners
+
+---
+
+## Security
+
+Helium 5G/ Mobile Hotspots must have several security features in order to participate in the Helium
+network and receive network rewards. These include a secure boot process and a secure element
+feature for managing the cryptographic keys which identify the Hotspot on the network.
+
+### Secure Boot
+
+Helium 5G/ Mobile Hotspots must only run authenticated firmware that has been provided by the
+manufacturer. This authentication process ensures that the Hotspot can be trusted by the rest of the
+network. This is especially important in the 5G network because Hotspots make observational reports
+on behalf of the entire network, such as:
+
+- User Equipment ("UE") attachment events
+- Signal strength (RSSI, RSRP, RSRQ)
+- Upstream bandwidth measurements
+
+Observational reports are ripe for “gaming” by the Hotspot owner because they directly impact the
+rewards that the Hotspot will receive.
+
+When designed and used correctly, a secure boot process can overcome these problems, leading to a
+more resilient and trustable network.
+
+### Secure Boot Requirements
+
+To be effective, the secure boot process you propose must meet these requirements:
+
+- There must be a secure hardware boundary encompassing:
+  - A boot CPU
+  - A one-time programmable memory area capable of storing:
+    - A trusted public key (or hash of such a key) for authenticating external boot code.
+    - A device-unique secret key
+  - A cryptographic hardware element capable of public key and private key cryptography.
+  - An unalterable (fused or masked) boot ROM.
+  - A static RAM.
+- The buss(es) within the secure hardware boundary must be protected.
+  - Activity on the bus cannot be inspected nor altered by any entity outside of the boundary.
+  - No outside entity shall be able to inspect the device-unique secret key in unencrypted format.
+  - No outside entity shall be able to alter the contents of the memory elements (SRAM, boot ROM,
+    OTP memory) inside the boundary.
+
+### Secure Update Process
+
+A manufacturer-controlled process encompassing:
+
+- The external boot code signing key (private portion).
+- A process for securely managing access to the signing key to trusted employees.
+
+---
+
+## Enhanced Secure Element
+
+Helium has always required a Secure Element for all Hotspots that participate in LoRaWAN (soon to be
+IOT) PoC rewards. This requirement will continue with the 5G network, however, the security
+requirements for 5G will be more strict than what is in place for LoRaWAN. To prevent confusion,
+these new requirements are known as **Enhanced Secure Element\***.\*
+
```

Comment:
`**Enhanced Secure Element\***.\*` rendered a little weird but I'm not sure about the intention

Reply:
good catch, notion to markdown copy paste artifact